projects/lib/src/board/westernboard.cpp

```diff
@@ -94,6 +95,13 @@ void WesternBoard::vInitialize()
 	m_rookOffsets[1] = -1;
 	m_rookOffsets[2] = 1;
 	m_rookOffsets[3] = m_arwidth;
+
+	m_pawnAmbiguous = pawnAmbiguity(StepType::Free);
+}
+
+inline int WesternBoard::getPawnPush(const PawnStep& ps, int sign) const
```

Comment:
https://isocpp.org/wiki/faq/inline-functions#inline-nonmember-fns

It should work as it is. The definition body would be needed in the header if more than one cpp-file referenced the function. Its a private member function for now. None of the other classes´ business.       

> "The reason you (almost always) put the definition (the {...} part) of an inline function in a header file is to avoid “unresolved external” errors from the linker. That error will occur if you put the inline function’s definition in a .cpp file and if that function is called from some other .cpp file."